src/server/game/Server/WorldSession.cpp

```diff
@@ -126,7 +126,7 @@ WorldSession::WorldSession(uint32 id, WorldSocket* sock, AccountTypes sec, uint8
         m_Address = sock->GetRemoteAddress();
         sock->AddReference();
         ResetTimeOutTime(false);
-        LoginDatabase.PExecute("UPDATE account SET online = %u WHERE id = %u;", realmID, GetAccountId());
+        LoginDatabase.PExecute("UPDATE account SET online = 1 WHERE id = %u;", GetAccountId());
```

Comment:
was this suppose to track in which realm the account is online?

Reply:
I don't think it serves to know in which realm the account is connected. But it's true we can keep that.
https://github.com/TrinityCore/TrinityCore/blob/master/src/server/game/Server/WorldSession.cpp#L146